plugin/pathvirt/pathvirt.cpp

```diff
@@ -363,6 +365,25 @@ dmtcp_event_hook(DmtcpEvent_t event, DmtcpEventData_t *data)
        }
        break;
     }
+    case DMTCP_EVENT_RESTART:
```

Comment:
Why do you need to add this event? It seems like what you want is already covered by:
a) call to `pathvirtInitialize()` from `dmtcp::callbackPostCheckpoint()` in `src/mtcpinterface.cpp`; and
b) cases for POST_EXEC and PRE_EXEC events in the pathvirt event hook function?